From a85b0b96c8ebe3e010d0d016cf88ab3c8bfc196a Mon Sep 17 00:00:00 2001
From: Victor Boctor <victor@mantishub.net>
Date: Sun, 12 Feb 2017 15:58:25 -0800
Subject: [PATCH] Sanitize window title

The window title is not sanitized.  That is not an issue when CSP is enable (default),
but if disabled, it can execute javascript that is set by a user who has access
to set configuration via Manage - Manage Configuration - Configuration Report page.

Fixes #22266
---
 core/layout_api.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/layout_api.php b/core/layout_api.php
index 6c648567a1..2648d8dc0a 100644
--- a/core/layout_api.php
+++ b/core/layout_api.php
@@ -415,7 +415,7 @@ function layout_navbar() {
 	echo '<div class="navbar-header">';
 	echo '<a href="' . $t_logo_url . '" class="navbar-brand">';
 	echo '<span class="smaller-75"> ';
-	echo config_get('window_title');
+	echo string_display_line( config_get('window_title') );
 	echo ' </span>';
 	echo '</a>';
 
